front-end/src/screens/TreasuryPost/query.ts

```diff
@@ -33,6 +33,7 @@ const treasuryProposalPost = gql`
         }
         content
         created_at
+        has_poll
```

Comment:
not needed
```suggestion
```

Reply:
This is needed for typescript to be happy when querying post.has_poll